external/dynamo/generalized/frontend.py

```
@@ -0,0 +1,434 @@
+
+            except HTTPException:
+                raise
+            except Exception as e:
+                logging.error("Error in chat completions: %s", e)
+                raise HTTPException(status_code=500, detail=str(e))
+
+        @self.app.get("/health")
+        async def health():
+            return {"status": "healthy"}
+
+    async def run_server(self, host: str = "0.0.0.0", port: int = 8099):
```

Ruff (0.14.8)

Possible binding to all interfaces

(S104)